packages/application-components/src/theming.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  transformTokensToCssVarsReferences,
+  designTokens as uiKitDesignTokens,
+} from '@commercetools-uikit/design-system';
+
+export const themesOverrides = {
+  default: {
+    colorForPageHeaderBottomBorder: uiKitDesignTokens.colorNeutral,
+    backgroundColorForMainPageDivider: uiKitDesignTokens.colorNeutral60,
+    backgroundColorForPageHeader: uiKitDesignTokens.colorNeutral95,
+    backgroundColorForTabularMainPageContent: uiKitDesignTokens.colorNeutral95,
+    borderBottomForTabularPageHeader: `1px solid ${uiKitDesignTokens.colorNeutral}`,
+    borderColorForDialogDivider: uiKitDesignTokens.colorNeutral,
+    borderColorForModalPageHeaderDivider: uiKitDesignTokens.colorNeutral60,
+    borderColorForModalTopBarWhenSurface: uiKitDesignTokens.colorNeutral,
+    heightForTab: '4px',
+    marginBottomForPageTopBar: uiKitDesignTokens.spacingS,
+    marginForModalPageHeader: `0 ${uiKitDesignTokens.spacingM}`,
+    marginLeftForModalPageHeaderControls: uiKitDesignTokens.spacingM,
+    marginTopForPageHeaderControls: uiKitDesignTokens.spacingS,
+    marginTopForPageSubtitle: uiKitDesignTokens.spacingM,
+    marginTopForTabControls: uiKitDesignTokens.spacingS,
+    paddingForDetailPageHeader: uiKitDesignTokens.spacingM,
+    paddingForDialogContent: `${uiKitDesignTokens.spacingM} 0 ${uiKitDesignTokens.spacingS}`,
+    paddingForMainPageHeader: uiKitDesignTokens.spacingM,
+    paddingForModalPageHeader: `${uiKitDesignTokens.spacingM} 0`,
+    paddingForModalTopBar: `${uiKitDesignTokens.spacingS} ${uiKitDesignTokens.spacingM}`,
+    paddingForPageContent: uiKitDesignTokens.spacingM,
+    paddingForTabularPageHeader: `${uiKitDesignTokens.spacingM} ${uiKitDesignTokens.spacingM} 0`,
+    paddingLeftForTabAsFirst: '0',
+    widthForDialogAsMedium: uiKitDesignTokens.constraint7,
+    widthForDialogAsLarge: uiKitDesignTokens.constraint10,
+  },
+  test: {
+    colorForPageHeaderBottomBorder: uiKitDesignTokens.colorNeutral90,
+    backgroundColorForMainPageDivider: uiKitDesignTokens.colorNeutral90,
+    backgroundColorForPageHeader: uiKitDesignTokens.colorSurface,
+    backgroundColorForTabularMainPageContent: uiKitDesignTokens.colorSurface,
+    borderBottomForTabularPageHeader: `1px solid ${uiKitDesignTokens.colorNeutral90}`,
+    borderColorForDialogDivider: uiKitDesignTokens.colorNeutral90,
+    borderColorForModalPageHeaderDivider: uiKitDesignTokens.colorNeutral90,
+    borderColorForModalTopBarWhenSurface: uiKitDesignTokens.colorSurface,
+    heightForTab: '2px',
+    marginBottomForPageTopBar: uiKitDesignTokens.spacingL,
+    marginForModalPageHeader: `0 40px`,
+    marginLeftForModalPageHeaderControls: uiKitDesignTokens.spacingXl,
+    marginTopForPageHeaderControls: uiKitDesignTokens.spacingS,
```

Comment:
Just wanted to double check - the value for the default theme and the new one are the same for this token

Reply:
That's a leftover, Thanks!

Removed in this commit: bfad34b266ca207bde3008b3cb0a77647d2ef128